downstream/modules/hub/proc-downloading-signature-public-keys.adoc

```diff
@@ -4,13 +4,15 @@
 
 = Downloading signature public keys
 
-After you sign and approve collections, download the signature public keys from the {HubName} UI. You must download the public key before you add it to the local system keyring.
+After you sign and approve collections, download the signature public keys from the {HubName} UI. 
+You must download the public key before you add it to the local system keyring.
 
 .Procedure
 
 . Log in to your private {HubName} instance in the {HubName} UI.
 
-. In the navigation pane, select menu:Signature Keys[]. The Signature Keys dashboard displays a list of multiple keys: collections and container images.
+. In the navigation pane, select menu:Signature Keys[]. 
```

Comment:
For accessibility, I think Donna has said we should use "From the navigation panel, select...". Donna has also pointed out that she doesn't see this selection in the UI, as per [PR #388](https://github.com/ansible/aap-docs/pull/388), so we need to verify if it is still valid.